packages/secret-numbers/.eslintrc.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
```

Comment:
I looked into normalizing this across all the packages.  That will end up being a project all on its own.  I choose to base it off the simplest one which was `ripple-address-codec`.

Reply:
👍